code-challenges.test.js

```diff
@@ -59,7 +78,25 @@ var hand2 = [5, 5, 3, 3, 4]
 // Expected output: false
 var hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
-
+describe("when an arrayOfNums is called", () => {
+  it("returns a boolean of true of false", () => {
+    boolean arrayOfNums = true;
```

Comment:
make sure to pseudocode first before putting down code. It's more important to understand the logic than to have working code. (But in this case, it's Java code so it wouldn't work within the Javascript environment 😉 )

Reply:
However, if you feel like the Java code is helping you understand a possible solution, comment it out and then pseudocode to confirm your logical understanding of what the code is doing